File: javascript-closure-angular-client/API/Client/IssConfByCity.js
goog.provide('API.Client.IssConfByCity');

/**
 * @record
 */
API.Client.IssConfByCity = function() {}

/**
 * City Code (IBGE)
 * @type {!number}
 * @export
 */
API.Client.IssConfByCity.prototype.cityCode;

/**
 * @type {!string}
 * @export
 */
API.Client.IssConfByCity.prototype.name;

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.IssConfByCity.prototype.state;

/**
 * @type {!boolean}
 * @export
 */
API.Client.IssConfByCity.prototype.issWhDestOtherCities;

/**
 * @type {!boolean}
 * @export
 */
API.Client.IssConfByCity.prototype.issWhDestSameCity;

/**
 * @type {!boolean}
 * @export
 */
API.Client.IssConfByCity.prototype.issWhOriginUnregSeller;

/**
 * @type {!Array<!API.Client.IssConfServiceList>}
 * @export
 */
API.Client.IssConfByCity.prototype.serviceList;

File: async-scala-client/src/main/scala/io/swagger/client/model/IssConfByCity.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class IssConfByCity (
  cityCode: Long,  // City Code (IBGE)
name: Option[String],
state: Option[StateEnum],
issWhDestOtherCities: Option[Boolean],
issWhDestSameCity: Option[Boolean],
issWhOriginUnregSeller: Option[Boolean],
serviceList: Option[List[IssConfServiceList]]
)
